app/assets/js/script.js

```diff
@@ -8,6 +8,32 @@
     var $doc = $(document);
     var $body = $(document.body);
 
+    var fixedElementOffset = $('.context-nav').offset().top;
+    var footerOffset = $('.SiteFooter').offset().top - 93;
+    var fixedElementHeight = $('.context-nav').height();
+
+    $(window).scroll(function (event) {
+      var y = $(this).scrollTop();
+
+      if (y < 148) {
+        $('.context-nav').css('top','1.3em');
```

Comment:
Let's cache all `$()` selectors outside the closure so we don't need to recompute them for every event triggered during scroll.